### DISCUSSION TITLE
CS%E5%9F%BA%E7%A1%80/%E5%AF%BC%E8%A7%88/

### DISCUSSION THREAD

**Discussion by giscus** [MingLLuo/CS-Plan#12] [Announcements]
# CS%E5%9F%BA%E7%A1%80/%E5%AF%BC%E8%A7%88/

https://cs-plan.com/CS%E5%9F%BA%E7%A1%80/%E5%AF%BC%E8%A7%88/

**Comment by KENNY-hub-vue** [comment]
请问java红宝书是叫什么名字啊，搜到了JavaScript的红宝书，但是Java的红宝书不确定叫什么名字。

**Reply by q3487633796** [reply]
可能是算法（第四版）
